app/utils/getCacheAnalysis.test.ts

```diff
@@ -1,9 +1,255 @@
+
+    const result = parseCacheStatus(cacheStatus)
+
+    expect(result).toHaveLength(3)
+    expect(result[0]?.cacheName).toBe('Netlify Edge') // Should be reversed per spec
+    expect(result[1]?.cacheName).toBe('Netlify Durable')
+    expect(result[2]?.cacheName).toBe('Next.js')
+    expect(result[0]?.parameters.fwd).toBe('miss')
+    expect(result[1]?.parameters.stored).toBe(true)
+    expect(result[2]?.parameters.hit).toBe(true)
+  })
+
+  it('parses cache status with numeric parameters', () => {
+    const cacheStatus = '"Netlify Edge"; hit; fwd-status=200; ttl=3600'
+
+    const result = parseCacheStatus(cacheStatus)
+
+    expect(result[0]?.parameters['fwd-status']).toBe(200)
+    expect(result[0]?.parameters.ttl).toBe(3600)
+  })
+
+  it('parses cache status with string parameters', () => {
+    const cacheStatus = '"Netlify Edge"; hit; key=cache-key-123; detail=some-detail'
+
+    const result = parseCacheStatus(cacheStatus)
+
+    expect(result[0]?.parameters.key).toBe('cache-key-123')
+    expect(result[0]?.parameters.detail).toBe('some-detail')
+  })
+
+  it('handles empty cache status string', () => {
+    const result = parseCacheStatus('')
+
+    expect(result).toEqual([])
+  })
+
+  it('ignores invalid cache status entries without parameters', () => {
+    const cacheStatus = '"Netlify Edge", "Valid Cache"; hit'
+
+    const result = parseCacheStatus(cacheStatus)
+
+    expect(result).toHaveLength(1)
+    expect(result[0]?.cacheName).toBe('Valid Cache')
+  })
+
+  it('ignores cache status entries with invalid parameters', () => {
+    const cacheStatus = '"Netlify Edge"; ; hit'
+
+    const result = parseCacheStatus(cacheStatus)
+
+    expect(result).toHaveLength(1)
+    expect(result[0]?.cacheName).toBe('Netlify Edge')
+    expect(result[0]?.parameters.hit).toBe(true)
+  })
+
+  it('sorts cache entries according to RFC 9211 precedence', () => {
+    const cacheStatus = '"Netlify Edge"; hit, "Next.js"; hit, "Netlify Durable"; hit'
+
+    const result = parseCacheStatus(cacheStatus)
+
+    // Should be sorted and then reversed
+    expect(result[0]?.cacheName).toBe('Netlify Edge')
+    expect(result[1]?.cacheName).toBe('Netlify Durable')
+    expect(result[2]?.cacheName).toBe('Next.js')
+  })
+
+  it.each([
+    { input: '"Cache"; fwd=bypass', expected: 'bypass' },
+    { input: '"Cache"; fwd=method', expected: 'method' },
+    { input: '"Cache"; fwd=uri-miss', expected: 'uri-miss' },
+    { input: '"Cache"; fwd=vary-miss', expected: 'vary-miss' },
+    { input: '"Cache"; fwd=miss', expected: 'miss' },
+    { input: '"Cache"; fwd=request', expected: 'request' },
+    { input: '"Cache"; fwd=stale', expected: 'stale' },
+    { input: '"Cache"; fwd=partial', expected: 'partial' },
+  ])('handles forward parameter $expected', ({ input, expected }) => {
+    const result = parseCacheStatus(input)
+    expect(result[0]?.parameters.fwd).toBe(expected)
+  })
```

Comment:
this probably doesn't need to be a table test. it's just threading the string as is? just test one string maybe.